### DISCUSSION TITLE
v0.6.1

### DISCUSSION THREAD

**Discussion by dail8859** [dail8859/NotepadNext#356] [New Releases]
## What's Changed
* Add encoding/decoding for Base 64 and URL
* Add menu option to remove empty links
* Add filename extension support on macOS
* Add "Show in Explorer" and "Open Command Prompt Here" for Windows
* Add functionality to combine search results on the same line
* Add message to find dialog when search wraps around
* Add Ukrainian translation
* Update Scintilla to v5.3.4
* Custom context menu for editors
* Identify more languages based on shebang
* Update desktop file to pass file name argument
* Provide better user error messages when files cannot be saved
* Prevent crash when opening non-readable file
* Fix SmartHighlighter stylers
* Fix loading of translation files
* Fix applying .editorconfig settings
* Fix regular expression searching when file contained 3 or 4 byte characters

**Full Changelog**: https://github.com/dail8859/NotepadNext/compare/v0.6...v0.6.1

<hr /><em>This discussion was created from the release <a href='https://github.com/dail8859/NotepadNext/releases/tag/v0.6.1'>v0.6.1</a>.</em>

**Comment by libaiyi888** [comment]
打不开，求解。

**Reply by dail8859** [reply]
Sorry I won't be able to help. There's no details to even begin to guess what's wrong.

**Reply by libaiyi888** [reply]
重装了系统。可以安装。现在打开提示找不到MSVCP140.dll，无法继续执行代码

**Reply by dail8859** [reply]
I would guess your computer is missing the latest Microsoft Visual C++ Redistributable, I'm assuming you'll need the x64 version.

https://learn.microsoft.com/en-US/cpp/windows/latest-supported-vc-redist?view=msvc-170

**Reply by libaiyi888** [reply]
是的。不过已经解决。谢谢。

**Comment by zliucd** [comment]
There is no Mac OSX arm64 package. Thanks.

**Reply by dail8859** [reply]
I'm not a Mac user, but according to https://github.com/dail8859/NotepadNext/issues/199 I thought it was was already supported?
